samples/create_cluster.js

```diff
@@ -0,0 +1,170 @@
+  checkOpStatus(client, opId)
+    .then(status => {
+      // success scenario
+      console.log(status);
+    })
+    .catch(status => {
+      // fail (not yet complete) scenario
+      if (retryCount < maxtries) {
+        console.log(`${status} will try after ${delay / 1000}s delay...`);
+        const newDelay = getFibonacciDelay(delay);
+        setTimeout(
+          () =>
+            checkStatusWithRetry(
+              client,
+              opId,
+              newDelay,
+              maxtries,
+              retryCount + 1
+            ),
+          delay
+        );
+      } else {
+        console.log(`${status} max retries reached, giving up.`);
+      }
+    });
+}
+
+/**
+ * This method creates a GKE cluster using the given client in the GCP project
+ * and zone indicated by the `gcpProject` and `gcpZone` parameters. The last
+ * argment is used as the name given to the newly created cluster.
+ *
+ * @param {container.v1.ClusterManagerClient} client the google cloud API client used to submit the request
+ * @param {string} gcpProject the GCP project where the cluster is to be created
+ * @param {string} gcpZone the GCP zone where the cluster is to be created
+ * @param {string} clusterName the name to be given to the new cluster
+ */
+async function createCluster(gcpZone, clusterName, networkName) {
+  // Create the Cluster Manager Client
+  const client = new container.v1.ClusterManagerClient();
+  const gcpProject = await client.getProjectId();
+  const clusterLocation = `projects/${gcpProject}/locations/${gcpZone}`;
+  const request = {
+    parent: clusterLocation,
+    cluster: {
+      name: clusterName,
+      network: networkName,
+      initialNodeCount: 2,
+      nodeConfig: {
+        machineType: 'e2-standard-2',
+      },
+    },
+  };
+  // invoke the create cluster API using the client
+  const [createOperation] = await client.createCluster(request);
+  // extract the unique identifier of the operation to checkback status
+  const opIdentifier = `${clusterLocation}/operations/${createOperation.name}`;
+  // schedule polling to check if the creation operation is completed
+  checkStatusWithRetry(client, opIdentifier, 1000, 20);
+}
+
+/**
+ * Entrypoint into the sample. Expects three arguments to the program.
+ * 1. The GCP Zone to use
+ * 2. The name to give to the new GKE cluster
+ *
+ * > node create_cluster.js --zone=<GCP_ZONE> --name=<CLUSTER_NAME>
+ */
+async function main() {
+  if (!args.name || !args.zone) {
+    console.log(
+      'Missing argument. Required name and zone (e.g. --name=gke-cluster, --zone=us-west1-a)'
+    );
+    exit(1);
+  }
+  // use the default network if no network is provided
+  let network = 'default';
+  if (args.network) {
+    network = args.network;
+  } else {
+    console.log(`No network provided; using ${network} network`);
+  }
+  // Create a new GKE cluster
+  createCluster(args.zone, args.name, network);
+}
+
+// program starts here
+main().catch(console.error);
```

Comment:
This should be: ```
process.on('unhandledRejection', (err) => {
  console.error(err.message);
  process.exitCode = 1;
});
main(...process.argv.slice(2));```

Reply:
Fixed in https://github.com/googleapis/nodejs-cloud-container/pull/519/commits/ee670d950b4f5a8e53ec494e4448b2bd2ca38e6b